### PR TITLE
Adds newline after unit-tests target.

### LINKED ISSUE #10836: Unable to run make unit-tests locally
```
$ make unit-tests
...
make: execvp: /bin/bash: Argument list too long
make: *** [Makefile:157: unit-tests] Error 127
```

### PR DESCRIPTION
**Pull request by Weil0ng**
It seems that in some env if the path is large enough for the full list
of files, the full bash command in that target gets too big for bash and
hence will trigger an error ``too many arguments``. (figured out by
@aanm and @joestringer )

Fixes: #10836

Signed-off-by: Weilong Cui <cuiwl@google.com>

### CODE REVIEW

**Comment by Weil0ng** [comment]
@joestringer @aanm

File: Makefile [hunk 1]
```diff
@@ -64,6 +64,11 @@ define generate_k8s_api_deepcopy
 	$(call generate_k8s_api,deepcopy,$(1),$(2))
 endef
 
+define NL
```
Comment (joestringer) [inline Makefile]: Please add a comment that describes what this is doing and why with a reference to the issue.

File: Makefile [hunk 2]
```diff
@@ -64,6 +64,11 @@ define generate_k8s_api_deepcopy
...
+
+
```
Comment (joestringer) [inline Makefile]: Are both newlines necessary?

**Review by joestringer — CHANGES_REQUESTED** [review]
Couple of minor nits.

**Comment by joestringer** [comment]
@Weil0ng for this issue since it fixes #10836, if you write "Fixes: #10836" in the PR or commit message then github will autoclose the other issue when this is merged.

File: Makefile (hunk 1 above)
Comment (Weil0ng) [reply Makefile]: Done.

File: Makefile (hunk 2 above)
Comment (Weil0ng) [reply Makefile]: Yup :)

**Comment by Weil0ng** [comment]
PTAL

**Review by joestringer — APPROVED** [review]

**Comment by joestringer** [comment]
test-me-please

**Comment by pchaigno** [comment]
I think the Travis CI failure is related:
```
go test -mod=vendor -ldflags "-X github.com/cilium/cilium/pkg/kvstore.consulDummyConfigFile=/tmp/cilium-consul-certs/cilium-consul.yaml -X github.com/cilium/cilium/pkg/testutils.CiliumRootDir=/home/travis/gopath/src/github.com/cilium/cilium -X github.com/cilium/cilium/pkg/datapath.DatapathSHA=1234567890abcdef7890 -X github.com/cilium/cilium/pkg/logging.DefaultLogLevelStr="error"" github.com/cilium/cilium/bugtool -test.v -timeout 360s -coverprofile=coverage.out -covermode=count -coverpkg ./... || exit 1; tail -n +2 coverage.out >> coverage-all-tmp.out;
/bin/bash: 2: unbound variable
Makefile:152: recipe for target 'unit-tests' failed
make: *** [unit-tests] Error 1
The command "./.travis/build.sh" exited with 2.
```
It was also failing with the same error on the previous build, for aarch64 only as well.

**Review by aanm — APPROVED** [review]

**Comment by aanm** [comment]
@Weil0ng it seems is a legit failure in travis:
```
go test -mod=vendor -ldflags "-X github.com/cilium/cilium/pkg/kvstore.consulDummyConfigFile=/tmp/cilium-consul-certs/cilium-consul.yaml -X github.com/cilium/cilium/pkg/testutils.CiliumRootDir=/home/travis/gopath/src/github.com/cilium/cilium -X github.com/cilium/cilium/pkg/datapath.DatapathSHA=1234567890abcdef7890 -X github.com/cilium/cilium/pkg/logging.DefaultLogLevelStr="error"" github.com/cilium/cilium/bugtool -test.v -timeout 360s -coverprofile=coverage.out -covermode=count -coverpkg ./... || exit 1; tail -n +2 coverage.out >> coverage-all-tmp.out;
/bin/bash: 2: unbound variable
Makefile:152: recipe for target 'unit-tests' failed
make: *** [unit-tests] Error 1
```

**Comment by Weil0ng** [comment]
PTAL. Have to use another workaround to cope with different versions of bash.

**Comment by joestringer** [comment]
Travis still seems unhappy, so I'll defer review until it is green.

**Review by joestringer — APPROVED** [review]
LGTM

**Comment by joestringer** [comment]
test-me-please

**Comment by Weil0ng** [comment]
hmm, that's odd, the failure seems like unrelated to this PR...

**Comment by joestringer** [comment]
@Weil0ng It may be. Usually we will try to look for issues with the test name and reference the issue so that we can track common flakes on master and make progress towards resolving them.

**Comment by pchaigno** [comment]
test-with-kernel

**Pull request merged**